js/components/faceted-datatable.html

```diff
@@ -1,46 +1,56 @@
-<div class="text-center" data-bind="if: $component.componentLoading(), css: {visible:componentLoading(), hidden: !componentLoading()}">
-	<h3><i class="fa fa-spinner fa-spin"></i> Loading ...</h3>
-</div>
-<div data-bind="ifnot: $component.componentLoading(), css: {visible: !componentLoading(), hidden: componentLoading()}">
-	<div class="row">
-		<div class="feFilter" data-bind="visible:$component.facets().length > 0, css: {'col-xs-2': $component.facets().length > 0}">
-			<div data-bind="with: $component.facets">
-				<div data-bind="foreach:$data">
-					<div class="facetName"><i class="fa fa-filter"></i><span data-bind="text:caption"></span></div>
-					<div class="facetScrollBox" data-bind="foreach:$data.facetItems">
-						<div class="facetMemberName" data-bind="click: function(data, event) { $component.updateFilters($data, event);}, css : {selected: selected}">
-							<span data-bind="attr: {id: $parentContext.$index() + '-' + $index()}, text:key + ' (' + value + ')'"></span>
-						</div>
-					</div>
-				</div>
-			</div>
-		</div>
-		<div data-bind="css: {'faceted-data-table-wrapper': true, 'col-xs-10': $component.facets().length > 0, 'col-xs-12': $component.facets().length == 0}">
-			<div class="facetedDataTable">
-				<table width="100%" class="conceptTable stripe compact hover" cellspacing="0" cellspacing="0" data-bind="dataTable:{
-							data: $component.tableData,
-							options: {
-								dom: $component.dom,
-								colVis: $component.colVis,
-								language: $component.language,
-								buttons: $component.buttons,
-								rowCallback: $component.rowCallback,
-								lengthMenu: $component.lengthMenu,
-								orderClasses: $component.orderClasses,
-								ordering: $component.ordering,
-								order: $component.order,
-								columns: $component.columns,
-								autoWidth: $component.autoWidth,
-								deferRender: $component.deferRender,
-								pageLength: $component.pageLength,
-								drawCallback: $component.drawCallback
-							},
-							scrollOptions: $component.scrollOptions,
-							onRowClick: $component.rowClick
-					} "><!-- ko if: $component.headersTemplateId -->
-					<thead data-bind="template: { name: $component.headersTemplateId }"></thead><!-- /ko -->
-				</table>
-			</div>
-		</div>
-	</div>
-</div>
+<div style="position: relative">
+	<div class="text-center" style="position: absolute; left: 50%; height: 50%; z-index: 100"
```

Comment:
- no inline styles
- make it half transparent - otherwise looks not so good
- move to separate component